### DISCUSSION TITLE
None of my stuff will load

### DISCUSSION THREAD

**Discussion by J-Hahn3156** [community/community#21345] [Other Feature Feedback, Questions, & Ideas]
Tried to use GitHub and nothing on my home page will load.  Can't even sign out.  I get this message: There was an error in loading the activity feed

**Comment by airtower-luna** [comment]
Do you have any add-on active that restricts which resources a website can load, or if it can run scripts? If so, you might need to adjust its settings.

**Comment by jfullstackdev** [comment]
also, try logging in in Incognito mode

**Comment by jmansilla-2014056** [comment]
1. Try using another browser.
2. Try clearing your browser's cache.
3. Try downloading the desktop version from github.
4. Try using another device.
5. Make sure you are registered with a valid email to which you have access, it is important that you check your email, sometimes for security reasons github notifies you of unusual activities.
